gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -707,9 +748,65 @@ def _is_node_order_in_spec_supported(self, context):
             raise InvalidNodeOrderInChain(
                     node_order=allowed_chain_combinations)
 
+    def _get_consumers_for_provider(self, context, provider):
+        '''
+        {
+            consuming_ptgs_details: [{'ptg': <>, 'subnets': <>}]
+            consuming_eps_details: []
+        }
+        '''
+
+        consuming_ptgs_details = []
+        consuming_eps_details = []
+
+        provided_prs_id = provider['provided_policy_rule_sets'][0]
```

Comment:
What if we have multiple prs, and first one is allow prs ?
We need consumers only corresponding to the redirect prs.